WMFData/Sources/WMFData/Resources/WMFData.xcdatamodeld/WMFData 5.xcdatamodel/contents

```diff
@@ -0,0 +1,60 @@
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="23788.4" systemVersion="24G231" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="CDArticleTab" representedClassName="CDArticleTab" syncable="YES" codeGenerationType="class">
+        <attribute name="identifier" attributeType="UUID" usesScalarValueType="NO"/>
+        <attribute name="isCurrent" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES"/>
+        <attribute name="timestamp" attributeType="Date" usesScalarValueType="NO"/>
+        <relationship name="items" optional="YES" toMany="YES" deletionRule="Nullify" ordered="YES" destinationEntity="CDArticleTabItem" inverseName="tab" inverseEntity="CDArticleTabItem"/>
+    </entity>
+    <entity name="CDArticleTabItem" representedClassName="CDArticleTabItem" syncable="YES" codeGenerationType="class">
+        <attribute name="identifier" attributeType="UUID" usesScalarValueType="NO"/>
+        <attribute name="isCurrent" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES"/>
+        <relationship name="page" maxCount="1" deletionRule="Nullify" destinationEntity="CDPage" inverseName="articleTabItems" inverseEntity="CDPage"/>
+        <relationship name="tab" maxCount="1" deletionRule="Nullify" destinationEntity="CDArticleTab" inverseName="items" inverseEntity="CDArticleTab"/>
+    </entity>
+    <entity name="CDCategory" representedClassName="CDCategory" syncable="YES" codeGenerationType="class">
+        <attribute name="projectID" attributeType="String"/>
+        <attribute name="title" attributeType="String"/>
+        <relationship name="pages" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="CDPage" inverseName="categories" inverseEntity="CDPage"/>
+    </entity>
+    <entity name="CDPage" representedClassName="CDPage" syncable="YES" codeGenerationType="class">
+        <attribute name="namespaceID" attributeType="Integer 16" defaultValueString="0" usesScalarValueType="YES"/>
+        <attribute name="projectID" attributeType="String"/>
+        <attribute name="timestamp" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="title" attributeType="String"/>
+        <relationship name="articleTabItems" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="CDArticleTabItem" inverseName="page" inverseEntity="CDArticleTabItem"/>
+        <relationship name="categories" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="CDCategory" inverseName="pages" inverseEntity="CDCategory"/>
+        <relationship name="pageViews" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="CDPageView" inverseName="page" inverseEntity="CDPageView"/>
+        <relationship name="savedDate" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="CDSavedPage" inverseName="page" inverseEntity="CDSavedPage"/>
```

Comment:
Maybe name this relationship savedPage? Feels odd to use `savedDate.savedDate` later on.